infrastructure/main.tf

```diff
@@ -151,7 +166,11 @@ module "claim-store-api" {
 
     // robot notifications
     RPA_NOTIFICATIONS_SENDER = "noreply@reform.hmcts.net"
-    RPA_NOTIFICATIONS_RECIPIENT = "${data.azurerm_key_vault_secret.rpa_email.value}"
+    RPA_NOTIFICATIONS_SEALEDCLAIMRECIPIENT = "${data.azurerm_key_vault_secret.rpa_email_sealed_claim.value}"
```

Comment:
Followed https://github.com/spring-projects/spring-boot/wiki/Relaxed-Binding-2.0 - kept it consistent with properties file and environment properties because we have camel case in emailProperties.java. Will it work even work, I doubt when _ or replaced with . dots.

Reply:
I check that locally, worked just fine.

Reply:
https://github.com/spring-projects/spring-boot/wiki/Relaxed-Binding-2.0#environment-variables.  They're not advocating such practices when it comes to environment properties. Why we're introducing such things? I disagree with such approach.

Reply:
You are right, we should not relay on binding between `x_y` and `xY` as this is not documented feature of the Spring Environment however here I am asking to have readable environment variables only.

I would be happy to compromise on something like `RPA_NOTIFICATIONS_SEALEDCLAIM_RECIPIENT`

Reply:
RPA_NOTIFICATIONS_SEALEDCLAIMRECIPIENT is readable enough. And as I said, it is not recommended and neither advisable to breakout properties as written in the note in the link provided.

Reply:
It is not. It is easily to make mistake while using something like `SEALEDCLAIMRECIPIENT`. 

Other thing is that the link provided does not advise how to structure properties in the application, it just explains how framework copes possible notations including ones with `-` and `_`.